### ISSUE TITLE
[FEATURE]

### ISSUE THREAD

**Issue by theobserver** [akasolace/HO#554]
HO is actively maintained now, yet people do not know about it. At the [Hattrick Wiki ](https://wiki.hattrick.org/wiki/Helper_application) it states that Hattrick Organizer last update was at 30. November 2009.

**We should inform the HO community that HO is alive and kicking** by adding the new HO active status in Hattrick Wiki .

Thank you for your time reading this.

**Comment by akasolace** [comment]
@theobserver yes please, updating the wiki would be welcome !

**Comment by akasolace** [comment]
@theobserver  is it user the user para_para ? I see the page on wiki has been recently been updated but there are still plenty of inaccurate informations. Let me know if I can help you in the process ...

**Comment by theobserver** [comment]
I do not see any update on the page. Please provide the link of the page you are looking at. Also ,at the link i provided at the start of this thread i have no writes to edit or i cannot find how i could get the rights to edit something in the hattrick wiki. Excuse my lame English. Regards.

**Comment by akasolace** [comment]
@theobserver  ok cool

> I do not see any update on the page.

If I look on https://wiki.hattrick.org/index.php?title=Hattrick_Organizer&action=history I see some changes on the 27 May and 14 June 2020 by user Para Para

**Comment by akasolace** [comment]
@akasolace I see it 3 ways

1) you update the wiki
2) in the wiki you simply point to the documentation in https://ho.readthedocs.io/en/latest/  and you help develop the documentation in readthedocs
3) we do both

I would have personally a preference for option 2) but happy to have your opinion on this,.

**Comment by theobserver** [comment]
There is much information for HO newcomers which can be re-used in your documentation [here](https://wiki.hattrick.org/wiki/Hattrick_Organizer) and [here](https://hattrick.fandom.com/wiki/Main_Page) 

I will see if i can bring it to your documentation and then point all links so there is a central place . Right now it is scattered all around the web from previous changes from developers and volunteers...

**Comment by akasolace** [comment]
yes ideally in the end all should be centrralized here https://ho.readthedocs.io/en/latest/

**Comment by theobserver** [comment]
Hi,
i edited some links to show to the new Ho and Ho manual. I also kept the old links mentioned above. Would you like me to add the above links to your manual too?

**Comment by akasolace** [comment]
@theobserver  thank you. What I would suggest is:
  
  - remove the [4]  and change the link from  https://wiki.hattrick.org/wiki/Hattrick_Organizer  to https://akasolace.github.io/HO/ 

  - https://wiki.hattrick.org/wiki/Hattrick_Organizer   make it a simple page which simply redirect to  https://akasolace.github.io/HO/    (in case someone endup there by google search)

   - add missing documentation in  https://ho.readthedocs.io/en/latest/  ... any help welcome with that


How does it sound ?

**Comment by akasolace** [comment]
@theobserver  can you give me your team ID in HT ?  thank you
